<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketUpdate</class>
 <widget class="QWidget" name="MarketUpdate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>508</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>在线更新</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_name">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:2px;
border:1px solid black;
padding:7px;</string>
       </property>
       <property name="text">
        <string>软件名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_localVersion">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:2px;
border:1px solid black;
padding:7px;</string>
       </property>
       <property name="text">
        <string>当前版本</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_updateTime">
       <property name="styleSheet">
        <string notr="true">border-radius:2px;
border:1px solid black;
padding:7px;</string>
       </property>
       <property name="text">
        <string>更新时间</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_newVersion">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius:2px;
border:1px solid black;
padding:7px;</string>
       </property>
       <property name="text">
        <string>更新版本</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="text_updateLog">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="button_check">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>检查更新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_update">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>安装更新</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
